hw2/05-form.js

```diff
@@ -1 +1,22 @@
-// Add your code here
+// Stuart Rimel
+
+const form = document.querySelector('form');
+
+form.addEventListener('submit', (event) => {
+  event.preventDefault();
+  const data = new FormData(event.target);
+  const progLang = data.get('programmingLang');
+  const opSys = data.get('operatingSystems');
+  const fullStack = data.get('fullStack');
+  console.log(`FORM SUBMITTED!
+    Details of submission:
+        Name: ${data.get('name')}
+        Email: ${data.get('email')}
+        Registration Status: ${data.get('status')}
+        Previous Classes:`);
```

Comment:
For logging out Previous classes, I think we can check if any of progLang, opSys, fullStack are chosen, then if at least one of them is chosen, we can print out this line. This will avoid the case where previous classes is printed out with no class

Reply:
That's a good idea, I re-worked that console log in the updated PR.